### DISCUSSION TITLE
Orientation of phone on telescope

### DISCUSSION THREAD

**Discussion by emailcraigs** [artyom-beilis/skyhopper#24] [General]
I'm trying to design a holder for my phone on a shorter telescope. I know that being parallel with the OTA is a must, but how mandatory is it for the phone to be sitting directly on the OTA itself? Can I have the phone slightly above the surface of the OTA and slightly to the left if it is still parallel to the viewing direction? I'm adding a pic for reference. Thanks!
![IMG_5529](https://github.com/artyom-beilis/skyhopper/assets/31557493/e7ff5f03-286c-431f-bb23-307967181b67)

**Comment by artyom-beilis** [comment]
Only direction is important. Shifting it to any direction not a problem

**Reply by bird0cosmic** [reply]
Does it work on equatorial mounts? If there are any issues i know of, i will let you know. The only mount i have is an equatorial mount.

**Reply by artyom-beilis** [reply]
> Does it work on equatorial mounts? If there are any issues i know of, i will let you know. The only mount i have is an equatorial mount.

No, but kinda yes.

It isn't designed for EQ mount because the axis of the phone must be aligned with the pointing direction of the OTA.

But I know some succeed with it using some extra DIY contraption.

Look here for some examples: 

https://www.cloudynights.com/topic/849553-astrohopper-107-released/?p=12307960
https://www.cloudynights.com/topic/911548-astrohopper-for-eq-mount/?p=14046166

**Comment by emailcraigs** [comment]
Thanks! Love your program! I'm hoping to get more accurate with it.
    On Friday, January 5, 2024 at 05:36:36 AM EST, Artyom Beilis ***@***.***> wrote:  
 
 


Only direction is important. Shifting it to any direction not a problem

**Comment by TOMPENTA** [comment]
Hello,
sky in HOPPER rotates too fast 1°/80s. The position of the objects does not match after a while.
                                     Best regards
                                                    Aquarius
https://astrofotky.cz/~aquarius

**Reply by artyom-beilis** [reply]
I don't really understand what do you mean? What the position of the phone what are you doing? Is it aligned or not?

**Comment by TOMPENTA** [comment]
I did this test:
1/ launched SKY Hopper
2/ placed the mobile phone in a stable vertical position on the table
3/ performed an Align to a nearby star
4/ watched the change of angle Az over time
5/ the stars should rotate at a rate of 1°/4 minutes

**Reply by artyom-beilis** [reply]
> 2/ placed the mobile phone in a stable vertical position on the table

Have you seen quick start guide? The phone should lie flat on the optical tube. If it stands vertically it points to zenith

![Orientation](https://raw.githubusercontent.com/artyom-beilis/skyhopper/main/images/qs_1_1.png)

**Reply by jroe1943** [reply]
Can I rotate the phone on the ota 90 deg to be inline with the eyepiece.  (I tried and it seems to give different results.)

**Comment by TOMPENTA** [comment]
I have been waiting for a Hopper type program for a long time. Thanks for him.
It's a shame that I only found out about him on the Czech astroforum now.

https://www.astro-forum.cz/viewtopic.php?t=34914

I'm a bit older now and I've been breaking my neck with a green laser, finder or collimator.
I believe that it will be possible to maintain the accuracy of Hopper's guidance even during a longer observation.

**Comment by TOMPENTA** [comment]
Hello,
I did several tests during the day on the position:
N:49°36 E18°21
Around noon, the rotation of the stars in Hooper slowed to 1°/140s.
At 20:30 UTC, Hooper shows Vega exiting the image  1°/60s.
In the eyepiece, of course, Vega travels 1°/240 s.
Therefore, Hooper is unusable after 60s after calibration.
For Hooper, there are only 6 hours in the day.
                                  Best regards
                                          Thomas

**Reply by artyom-beilis** [reply]
Do you expect the phone gyro to follow the earth rotation? It isn't going to happen. It isn't just accurate enough.

You probably came here with wrong expectations.

The idea is you find a nearby star you can identify, align and than jump to the target. You are expected to align frequently by design. And by design realigning is trivial task.

I suggest go over quick start manual once again. 

It is much closer to start hopping assistant that measures movement between known position to the target rather than push to system that you align once and forget.

**Comment by TOMPENTA** [comment]
For a programmer, user feedback should be the most necessary thing he can wish for.
You still don't even think about the fact that you might have a fundamental error in the software.

I'll explain it from the other end:
1/ For Hooper to work, you must have your own small planetarium that calculates the positions of the stars according to UTC and the position of the observer.
2/ According to the data of the mobile phone sensors (compass+tilts), it will show the direction to the desired object after calibration.
3/ If the planetarium and sensors with random error are working properly, one calibration is enough.
4/ The compass error on my mobile phone is about +-1°. When using the average of several values, it comes to +- 0.2°.
5/ Tilt error is better than 0.1°.

Hopper's problem is that his planetarium spins too fast.
Very soon after calibration, the position of the object in Hooper does not match the actual position of the object.

Have you tested your own software? Have astronomers in different countries tested it? Already three years ago, Hooper's inaccuracies were discussed on the American forum.
                                                      Best regards
                                                                  Thomas

**Reply by artyom-beilis** [reply]
> Have you tested your own software? Have astronomers in different countries tested it? Already three years ago, Hooper's 

Obviously yes. And by the way it is called AstroHopper and it is widely popular. It has around several thousandth users around the world and 1/3 of dobsonian users in my local astro community use one.

> The compass error on my mobile phone is about +-1°. When using the average of several values, it comes to +- 0.2°.

1st of all you are wrong, magnetic compass are almost never this accurate especially on phones (that can have an error of 10-20 degrees easily) plus averaging wouldn't help because compass typically has systematic and not a random error.

That is why after alignment AstroHopper uses motion/gyro sensors and not magnetic ones.

> Tilt error is better than 0.1°.

This is also inaccurate, absolute error can be withing several degrees (there is a paper that measured it) however Relative error is small that is why I can use it.

Finally some phones have better sensors some horrible. My phone/sensors sometimes just start accidentally drifting the sensors in smart phones are far from perfect but generally good enough to bring you close to the target after a short hop.

Once again. I don't know what the phone you use, I still don't understand what exactly is happening and what do you mean by  moves too fast. Have you checked other phones?

Look at this video (not mine): https://www.youtube.com/watch?v=6-_58mSGz1Q maybe you'll see what is going on and how to use it.

It may be

1. improper expectations
2. improper use/alignment
3. poor sensors accuracy

Or maybe some other issue - but I still don't understand what exactly you are trying to say.

**Comment by TOMPENTA** [comment]
We still don't understand, please accept for a moment that you have a software bug. :-)
I will describe how ASTROHOPPER (sorry) behaves on my dobson and Samsung Galaxy A53.

1/ I move the viewfinder to Vega.
2/ Vega is in the center of the eyepiece.
3/ I will align to Vega
4/ I immediately enter "search" for Vega
5/ While Vega moves a little in the eyepiece, ASTROHOPPER reports after a minute that I should move the telescope by a full 1°.
ASTROHOPPER thinks there are 6 hours in the day and his planetarium spins too fast.
Try it on your telescope.

Daytime Dobsonian exam:
1/ Alignment to a distant object. I remember Az and Alt.
2/ Rapid movement of the telescope by 90°/90° and immediate return to the object.
3/ Az and Alt are the same.
4/ Wait 1 minute.
5/ Repeat the movement and back.
6/ Alt same, but Az moves by 1°/60s.

                                               Best regards
                                                    Thomas

**Reply by artyom-beilis** [reply]
Got your point.

I did several checks - browser no sensor change aligned on several targets compared drift to stellarium same same.

Too real phone pointed to Regulus (~50 degrees alt close to E) compared alt-az to stellarium.

Alt change was exact - 1 degree in stellarium to 1 degree in astrohopper - in same time period. The az drift was smaller 0.9 degree instead of 1.4 degree - but I suspect some vibrations maybe gyro drift can't tell exactly without proper debugging.

But I must note you that Alt-az coordinates do not change with 15 deg per hour. Actually Az can change very fast - when object close to zenith (consider moving E to W over zenith change by 180 degrees)

So I don't see an error currently. At least not x4 times faster - maybe somewhat slower on azimuth but it can also be related to gyro drift. Can't tell exactly.

**Comment by TOMPENTA** [comment]
Here we say: "like throwing peas against the wall" :-)
Unfortunately, ASTROHOPPER does not work in our country.
                                              Goodbye
                                                       Thomas
